src/loader.cr

```diff
@@ -2,8 +2,10 @@ require "./loader/*"
 
 path = ARGV[0]
 
-if ARGV[1]?
-  path = File.expand_path(File.join(ARGV[1], path))
+unless path == File.expand_path(path)
+  path = File.join(ARGV[1], path)
 end
 
+path = File.expand_path(path)
+
```

Comment:
You can replace all of this with `path = File.expand_path(ARGV[0], ARGV[1])`.

Reply:
Lovely! Did not check api of that function :)